libraries/extensions/telemetry/metrics/src/lib.rs

```diff
@@ -10,33 +10,29 @@
 //! [`sysinfo`]: https://github.com/GuillaumeGomez/sysinfo
 //! [`opentelemetry-rust`]: https://github.com/open-telemetry/opentelemetry-rust
 
-use futures::stream::Stream;
-use futures::StreamExt;
-use opentelemetry::sdk::metrics::{selectors, PushController};
-use opentelemetry_otlp::{ExportConfig, WithExportConfig};
-use std::time::Duration;
-
-// Skip first immediate tick from tokio, not needed for async_std.
-fn delayed_interval(duration: Duration) -> impl Stream<Item = tokio::time::Instant> {
-    opentelemetry::util::tokio_interval_stream(duration).skip(1)
-}
+use opentelemetry::metrics::{self};
+use opentelemetry_sdk::{metrics::MeterProvider, runtime};
 
+use opentelemetry_otlp::{ExportConfig, WithExportConfig};
 /// Init opentelemetry meter
 ///
 /// Use the default Opentelemetry exporter with default config
 /// TODO: Make Opentelemetry configurable
 ///
-pub fn init_meter() -> PushController {
-    let export_config = ExportConfig::default();
+pub fn init_metrics() -> metrics::Result<MeterProvider> {
+    let endpoint = std::env::var("OTEL_EXPORTER_OTLP_METRICS_ENDPOINT")
```

Comment:
What is the reason for using this new environment variable?

Reply:
Yeah so basicallly, you could have a specific port, specific HOST address. See: https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/#otel_exporter_otlp_metrics_endpoint

Currently, I'm using telegraf at: localhost:4317 (default) but there is no reason for this to always be true.

Reply:
Makes sense. Looks like the `opentelemetry-otlp` crate already uses some [similar env variables](https://docs.rs/opentelemetry-otlp/latest/src/opentelemetry_otlp/exporter/mod.rs.html#129-130) by default. I wonder if it makes sense to use the same ones for consistency?

Reply:
Thanks for clarifying! I just saw that the official docs mention this environment variable as well: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#example-2